dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/eutaxonomy/financials/EuTaxonomyDataForFinancials.kt

```diff
@@ -0,0 +1,33 @@
+package org.dataland.datalandbackend.model.eutaxonomy.financials
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import org.dataland.datalandbackend.annotations.DataType
+import org.dataland.datalandbackend.model.enums.eutaxonomy.AttestationOptions
+import org.dataland.datalandbackend.model.enums.eutaxonomy.YesNo
+import org.dataland.datalandbackend.model.enums.eutaxonomy.financials.FinancialServicesType
+import java.util.EnumSet
+
+/**
+ * --- API model ---
+ * Fields of the questionnaire for the EuTaxonomyForFinancials framework
+ */
+@DataType
+data class EuTaxonomyDataForFinancials(
```

Comment:
Der name des DataTypes (in den Requests, usw..) sollte dem Pfad in Gewisser Weise entsprechen. Also Hier demnach EutaxonomyForFinancials (sodass man dass automatisch übersetzen kann und kein Mapping braucht). Kann aber meinetwegen auch in einem seperaten Ticket geschehen

Reply:
Wird mit DALA-381 geschlossen